GTMAppAuth/Sources/macOS/GTMKeychain_macOS.m

```diff
@@ -108,6 +106,12 @@ + (NSData *)passwordDataFromKeychainForName:(NSString *)keychainItemName {
 }
 
 + (BOOL)removePasswordFromKeychainForName:(NSString *)keychainItemName {
+  NSString *prefKey = [self prefsKeyForName:keychainItemName];
+  NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
+  if (![defaults boolForKey:prefKey]) {
```

Comment:
Might add a comment here explaining why we do this, similar to the one in `passwordFromKeychainForName:`.

Reply:
Done.